Index: knn/knn_ocr.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/knn/knn_ocr.py b/knn/knn_ocr.py
--- a/knn/knn_ocr.py	
+++ b/knn/knn_ocr.py	
@@ -5,8 +5,8 @@
 
 def knn_ocr_normal(test):
     # 训练knn模型
-    samples = np.load('samples.npy')
-    labels = np.load('label.npy')
+    samples = np.load('./knn/samples.npy')
+    labels = np.load('./knn/label.npy')
 
     knn = cv2.ml.KNearest_create()
     knn.train(samples, cv2.ml.ROW_SAMPLE, labels)
